biopet-core/src/test/scala/nl/lumc/sasc/biopet/core/report/MultisampleReportBuilderTest.scala

```diff
@@ -41,10 +45,25 @@ class MultisampleReportBuilderTest extends TestNGSuite with Matchers {
 
       def libraryPage(sampleId: Int, libraryId: Int, args: Map[String, Any]) = ReportPage(Nil, Nil, Map())
     }
+    val dbFile = File.createTempFile("summary.", ".db")
+    dbFile.deleteOnExit()
+    val db = SummaryDb.openSqliteSummary(dbFile)
+    db.createTables()
+
+    val sample = Some("sampleName")
+    val lib = Some("libName")
+
+    sample.foreach { sampleName =>
+      val sampleId = Await.result(db.createSample(sampleName, 0), Duration.Inf)
```

Comment:
Hmm..you can try this on your own intepreter: compare the execution time of the two code blocks below.

(but first, some boilerplate)
```scala
import scala.concurrent._
import scala.concurrent.duration._
implicit def ec = ExceutionContext.global._
```

Now for the `Future` function that has a noticeable delay:
```scala
def func(n: int) = Future { Thread.sleep(3000); n * 2 }
```

Execution with the awat inside `.foreach` is something like this:
```scala
Seq(1, 2, 300).foreach { num =>
   val result = Await.result(func1(num), 20.seconds)
   println(result)
}
```

And with `Await` launched just once:
```scala
val futures = Seq(1, 2, 300).map { func1 }
val future = Future.sequence(futures)
val result = Await.result(future, 20.seconds)
result.foreach { println }
```

The first block should take roughly about 9 seconds (since `func1` takes 3 seconds to complete and we have 3 invocations). But the second one should only take a little more than 3 seconds, since we only do the `Await` call once.